product/src/main/resources/application.yml

```
@@ -1,6 +1,8 @@
 spring:
   application:
     name: product-service
+  config:
+    import : optional:configserver:http://localhost:8888
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)